src/main/java/model/input/file/DataFile.java

```diff
@@ -25,18 +34,22 @@
 	private List<Class<? extends DataValue>> columnList;
 	private boolean firstRowAsHeader;
 	private Class[] columnTypes;
-	
+	private DataTableBuilder builder = new DataTableBuilder();
+	private boolean hasMetaData;
+
 	/**
 	 * Creates a new type of a DataFile. Sets the default range of lines to read
 	 * from 1 to integer maxvalue.
 	 * @param path The path to the DataFile
+	 * @throws FileNotFoundException When the file can not be found
 	 */
-	public DataFile(String path) {
+	public DataFile(String path) throws FileNotFoundException {
```

Comment:
Het is niet mooi dat de constructor al IO gaat doen, terwijl je createDataTable dat ook al doet. Waarom moet hij een FileNotFoundException gooien?